lib/browser/api/extensions.js

```diff
@@ -538,7 +538,13 @@ ipcMain.on('chrome-cookies-getall', function (evt, responseId, details) {
 ipcMain.on('chrome-cookies-get', function (evt, responseId, details) {
   const cb = (error, cookie) => {
     if (!evt.sender.isDestroyed()) {
-      evt.sender.send('chrome-cookies-get-response-' + responseId, error, cookie)
+      var first = cookie
```

Comment:
@jonathansampson that is what you'd think based on the name, but that is not how they are implemented https://github.com/brave/muon/blob/master/atom/browser/api/atom_api_cookies.cc#L223